src/main/java/commands/AddDeadlineCommand.java

```diff
@@ -0,0 +1,52 @@
+package commands;
+import java.time.format.DateTimeParseException;
+
+import exceptions.MorganException;
+import tasks.DeadlineTask;
+
+
+/**
+ * This is an AddDeadlineCommand Class, which inherits from AddCommand.
+ * The execution of this command will add a deadline task to the task list.
+ */
+public class AddDeadlineCommand extends AddCommand {
+    public static final String KEYWORD = "deadline";
```

Comment:
I like how you used Caps letters for final constants in your class. Perhaps you could you could consider spacing out your constants by the type of operations that they do?